pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -312,6 +423,10 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.unsure
+                    
+        # # UNKNOWN
+        # unknown='unknown'                               # An unitialized value
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?